yapftests/yapf_test.py

```diff
@@ -15,6 +15,7 @@
 """Tests for yapf.yapf."""
 
 import io
+from re import match
```

Comment:
I prefer to use `import re` and use `re.match(...)` instead of importing `match`.